src/components/ListDetail/ListDetail.tsx

```diff
@@ -86,6 +86,7 @@ export const ListDetail = ({
   overflow,
   onChange,
   required,
+  title,
```

Comment:
I don't see this one being used either.

Reply:
... as a matter of fact... now that you mention it... <scratchesHead/> ... no... taking another look to see why they quieted TS